### PR TITLE
New django cheat sheet.

### PR DESCRIPTION
**Pull request by pjhampton**
### Description
Django is a web framework for perfectionists with deadlines. This cheat sheet gives an overview of the various command-line tools + django internals.

### Data Source
https://www.djangoproject.com/

### Example Queries
django cheat sheet

### Triggers
No triggers applied.

### Screenshots

screen shot 2016-03-19 at 13 26 33
screen shot 2016-03-19 at 13 26 21

### References
https://duck.co/ia/view/django_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/django.json [hunk 1]
```diff
@@ -0,0 +1,93 @@
+{
+    "id": "django_cheat_sheet",
+    "name": "Django Cheat Sheet",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/django.json]: @pjhampton Lets remove "Cheat Sheet" from name

File: share/goodie/cheat_sheets/json/django.json [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+        "Starting django",
+        "Database Operations",
+        "App Files"
+    ],
+
+    "sections": {
+        "Starting django": [
+            {
+                "key": "$ pip install django",
+                "val": "Installs django from PyPI."
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/django.json]: @pjhampton Avoid periods at end of val fields and description

File: share/goodie/cheat_sheets/json/django.json [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+    "section_order": [
+        "Starting django",
+        "Database Operations",
+        "App Files"
+    ],
+
+    "sections": {
+        "Starting django": [
+            {
+                "key": "$ pip install django",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/django.json]: Let's remove the `$` -- we've chose to keep that out from other terminal IAs

Comment (pjhampton) [inline share/goodie/cheat_sheets/json/django.json]: @moollaza done! :smile:

**Comment by tagawa** [comment]
Note: This PR seems to have overwritten #2730.

**Pull request closed without merging**